src/app/hooks/useOptimizelyScrollDepth/index.jsx

```diff
@@ -22,9 +20,7 @@ const useOptimizelyScrollDepth = () => {
   const [scrollHundred, setScrollHundred] = useState(false);
 
   const experimentVariation = useOptimizelyVariation(OPTIMIZELY_CONFIG.flagKey);
-  const hasVariationKey = experimentVariation !== null;
-
-  const sendScrollEvents = hasVariationKey && !isAmp;
+  const sendScrollEvents = experimentVariation !== null;
 
   useEffect(() => {
     if (!sendScrollEvents) {
```

Comment:
Can we remove `sendScrollEvents` and just use 
```suggestion
  useEffect(() => {
    if (!experimentVariation) {
```